sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeBeforeApiSuccessTest.java

```diff
@@ -445,6 +467,24 @@ public void happyPathUntilHangingCallWithToken() throws Exception {
         when(UUIDUtils.randomUUID()).thenReturn(requestId);
         when(PreferencesStorage.getString(PREFERENCE_KEY_REQUEST_ID)).thenReturn(requestId.toString());
 
+        /* Mock install id from AppCenter */
+        final UUID installId = UUID.randomUUID();
+        when(AppCenter.getInstallId()).thenReturn(new AppCenterFuture<UUID>() {
+            @Override
```

Comment:
add blank line, prefer using mocking framework than using anonymous inner classes. See all the test code for many examples.

Reply:
Fixed that, it was looking too ugly, agree.